src/codegen/function_body_chunk.rs

```diff
@@ -37,6 +37,13 @@ struct ReturnValue {
     pub ret: return_value::Info,
 }
 
+fn change_name(s: &str) -> String {
```

Comment:
Why don't use `nameutil::mangle_keywords` instead?

Reply:
Yes, better to use the existing function.

Reply:
I didn't know we had it! Makes my life simpler. :)